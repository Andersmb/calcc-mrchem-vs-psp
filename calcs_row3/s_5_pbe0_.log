Starting job 5688409 on c1-[24,36,38],c2-49,c3-18,c5-49 at Fri May 20 18:16:33 CEST 2022

Copying s_5_pbe0_.out to /cluster/home/ambr/projects/psp/row3/calcs/doublets
Copying s_5_pbe0_.json to /cluster/home/ambr/projects/psp/row3/calcs/doublets

Task and CPU usage stats:
JobID           JobName  AllocCPUS   NTasks     MinCPU MinCPUTask     AveCPU    Elapsed ExitCode 
------------ ---------- ---------- -------- ---------- ---------- ---------- ---------- -------- 
5688409       s_5_pbe0_        180                                             00:28:06      0:0 
5688409.bat+      batch         36        1   00:00:02          0   00:00:02   00:28:06      0:0 
5688409.ext+     extern        180        6   00:00:00          0   00:00:00   00:28:06      0:0 
5688409.0      mrchem.x        180       30   00:27:34         29   01:13:53   00:27:43      0:0 

Memory usage stats:
JobID            MaxRSS MaxRSSTask     AveRSS MaxPages   MaxPagesTask   AvePages 
------------ ---------- ---------- ---------- -------- -------------- ---------- 
5688409                                                                          
5688409.bat+     19336K          0     19336K        0              0          0 
5688409.ext+          0          0          0        0              0          0 
5688409.0      8463968K         29 3751909922        0             11          0 

Disk usage stats:
JobID         MaxDiskRead MaxDiskReadTask    AveDiskRead MaxDiskWrite MaxDiskWriteTask   AveDiskWrite 
------------ ------------ --------------- -------------- ------------ ---------------- -------------- 
5688409                                                                                               
5688409.bat+       19.44M               0         19.44M        0.07M                0          0.07M 
5688409.ext+        0.00M               0          0.00M            0                0              0 
5688409.0           6.77M               0          6.74M       67.01M                8         42.33M 

Job 5688409 completed at Fri May 20 18:44:40 CEST 2022
