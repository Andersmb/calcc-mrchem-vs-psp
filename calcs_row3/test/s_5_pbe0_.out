                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         HEAD                                     ***
***         Git commit hash    e7d4b230bb6df8495b9a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Sat Mar 19 14:14:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :        (1 bank)                              30
 OpenMP threads          :                                               6
 Total cores             :                                             175
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (6 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               2
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       S            :        0.000000        0.000000       -1.124207
    1       F            :        0.000000        0.000000        1.897465
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000       -0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-04
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                              13
 Beta electrons          :                                              12
 Total electrons         :                                              25
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              9.999999542625e-01
    1    1    a          :                              9.999998929672e-01
    2    1    a          :                              9.999998400793e-01
    3    1    a          :                              1.000000097772e+00
    4    1    a          :                              1.000000001071e+00
    5    1    a          :                              1.000000003899e+00
    6    1    a          :                              1.000000552422e+00
    7    1    a          :                              1.000005397945e+00
    8    1    a          :                              9.999931086724e-01
    9    1    a          :                              1.000000237035e+00
   10    1    a          :                              1.000000368477e+00
   11    1    a          :                              9.999995301495e-01
   12    1    a          :                              9.999995614306e-01
   13    1    b          :                              9.999998895875e-01
   14    1    b          :                              9.999999496726e-01
   15    1    b          :                              9.999998347492e-01
   16    1    b          :                              1.000000097877e+00
   17    1    b          :                              1.000000001267e+00
   18    1    b          :                              1.000000003215e+00
   19    1    b          :                              1.000000554947e+00
   20    1    b          :                              1.000005399386e+00
   21    1    b          :                              9.999931087851e-01
   22    1    b          :                              1.000000237037e+00
   23    1    b          :                              1.000000368387e+00
   24    1    b          :                              9.999995374865e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Precision               : 1.00000e-04
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                493.661110064910
 E-N energy              :            (au)              -1274.697709739948
 Coulomb energy          :            (au)                274.063902068756
 Exchange energy         :            (au)                 -8.764426888307
 X-C energy              :            (au)                -27.103196220373
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 47.655737241642
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -542.840320714962
 Nuclear energy          :            (au)                 47.655737241642
---------------------------------------------------------------------------
 Total energy            :            (au)             -4.951845834733e+02
                         :      (kcal/mol)             -3.107330150323e+05
                         :        (kJ/mol)             -1.300106935739e+06
                         :            (eV)             -1.347465837172e+04
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -88.514600670555
    1    1    a          :            (au)                -24.571946365619
    2    1    a          :            (au)                 -8.041916941583
    3    1    a          :            (au)                 -5.989225368353
    4    1    a          :            (au)                 -5.988000464038
    5    1    a          :            (au)                 -5.969029348326
    6    1    a          :            (au)                 -1.230165539324
    7    1    a          :            (au)                 -0.727943954422
    8    1    a          :            (au)                 -0.465857230509
    9    1    a          :            (au)                 -0.463180605946
   10    1    a          :            (au)                 -0.438030561732
   11    1    a          :            (au)                 -0.298539250331
   12    1    a          :            (au)                 -0.261133186425
   13    1    b          :            (au)                -88.509783771383
   14    1    b          :            (au)                -24.566533027197
   15    1    b          :            (au)                 -8.036872411929
   16    1    b          :            (au)                 -5.985498947029
   17    1    b          :            (au)                 -5.971372146380
   18    1    b          :            (au)                 -5.966399009902
   19    1    b          :            (au)                 -1.215626515222
   20    1    b          :            (au)                 -0.690816338119
   21    1    b          :            (au)                 -0.450837990043
   22    1    b          :            (au)                 -0.430697672121
   23    1    b          :            (au)                 -0.408554902611
   24    1    b          :            (au)                 -0.248316003271
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -285.440878222371
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 200
 KAIN solver             : 10
 Localization            : First two iterations
 Diagonalization         : Off
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : 1.00000e-06
 Orbital threshold       : 5.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          5.000000e+00        -495.184583473320         -4.951846e+02
    1          2.495325e-01        -497.696820200472         -2.512237e+00
    2          9.866391e-02        -497.781288524183         -8.446832e-02
    3          6.232601e-02        -497.795553605954         -1.426508e-02
    4          3.497688e-02        -497.799750093977         -4.196488e-03
    5          4.916921e-02        -497.803231836516         -3.481743e-03
    6          1.710305e-02        -497.803796047769         -5.642113e-04
    7          2.292200e-02        -497.804380979736         -5.849320e-04
    8          1.020378e-02        -497.804484053629         -1.030739e-04
    9          1.196934e-02        -497.804568519281         -8.446565e-05
   10          4.352483e-03        -497.804603662755         -3.514347e-05
   11          4.549734e-03        -497.804630403948         -2.674119e-05
   12          2.001153e-03        -497.804635207130         -4.803183e-06
   13          1.324169e-03        -497.804637479324         -2.272193e-06
   14          1.703647e-03        -497.804639926751         -2.447427e-06
   15          9.091562e-04        -497.804640850717         -9.239664e-07
   16          5.977160e-04        -497.804641240453         -3.897360e-07
   17          4.282314e-04        -497.804641428083         -1.876294e-07
   18          3.780615e-04        -497.804641592884         -1.648018e-07
   19          2.497780e-04        -497.804641649128         -5.624327e-08
   20          1.238796e-04        -497.804641669752         -2.062484e-08
---------------------------------------------------------------------------
                      SCF converged in 20 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               2
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       S            :        0.000000        0.000000       -1.124207
    1       F            :        0.000000        0.000000        1.897465
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000       -0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                496.999176220724
 E-N energy              :            (au)              -1280.563902857569
 Coulomb energy          :            (au)                273.975455234573
 Exchange energy         :            (au)                 -8.758543060591
 X-C energy              :            (au)                -27.112564448531
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 47.655737241642
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -545.460378911394
 Nuclear energy          :            (au)                 47.655737241642
---------------------------------------------------------------------------
 Total energy            :            (au)             -4.978046416698e+02
                         :      (kcal/mol)             -3.123771263599e+05
                         :        (kJ/mol)             -1.306985897538e+06
                         :            (eV)             -1.354595378415e+04
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -89.116046513400
    1    1    a          :            (au)                -24.824072399781
    2    1    a          :            (au)                 -8.088964304836
    3    1    a          :            (au)                 -6.031040531196
    4    1    a          :            (au)                 -6.027256296441
    5    1    a          :            (au)                 -6.012422157346
    6    1    a          :            (au)                 -1.285191323021
    7    1    a          :            (au)                 -0.755909647093
    8    1    a          :            (au)                 -0.527018313814
    9    1    a          :            (au)                 -0.526613456396
   10    1    a          :            (au)                 -0.509957254592
   11    1    a          :            (au)                 -0.326640224950
   12    1    a          :            (au)                 -0.280574755557
   13    1    b          :            (au)                -89.110365749707
   14    1    b          :            (au)                -24.819965588390
   15    1    b          :            (au)                 -8.083406768523
   16    1    b          :            (au)                 -6.024289109630
   17    1    b          :            (au)                 -6.010930319304
   18    1    b          :            (au)                 -6.009316121997
   19    1    b          :            (au)                 -1.276021255953
   20    1    b          :            (au)                 -0.719193126760
   21    1    b          :            (au)                 -0.508639645916
   22    1    b          :            (au)                 -0.506046490103
   23    1    b          :            (au)                 -0.493630341419
   24    1    b          :            (au)                 -0.268452721997
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -288.141964418121
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :        0.000007       -0.000004        0.627922
 Magnitude               :            (au)                        0.627922
                         :         (Debye)                        1.596018
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.000000       -0.910119
 Magnitude               :            (au)                        0.910119
                         :         (Debye)                        2.313291
---------------------------------------------------------------------------
 Total vector            :        0.000007       -0.000004       -0.282197
 Magnitude               :            (au)                        0.282197
                         :         (Debye)                        0.717273
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h 27m 34s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
